appveyor.yml

```diff
@@ -9,6 +9,7 @@ install:
   - rustup-init.exe -y --default-host x86_64-pc-windows-msvc --default-toolchain nightly
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
   - if defined OTHER_TARGET rustup target add %OTHER_TARGET%
+  - rustup component add clippy || exit 0
```

Comment:
I believe this means if Clippy isn't available, it will stop the `install` step (without any errors), and never initialize the submodules. Is that intended? Should this instead be `rustup component add clippy || true`?

Reply:
I did some tests previously (just did another: https://ci.appveyor.com/project/ehuss/test/builds/24512814).  Appveyor seems to treat each step independently, so either `exit 0` or `true` should work.  

I never noticed the submodule update, I have no idea why it is there, we don't have submodules.

Reply:
Ah got it. That indeed seems different from Travis (from what I remember).

> I never noticed the submodule update, I have no idea why it is there, we don't have submodules.

Indeed. 🤔 It's also not in the `.travis.yml`, so seems safe to remove?